hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/queries/AsNodeOperatorQueriesTest.java

```diff
@@ -0,0 +1,144 @@
+import static com.hedera.services.bdd.spec.HapiSpec.hapiTest;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountBalance;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountInfo;
+import static com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sleepFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+
+import com.hedera.services.bdd.junit.HapiTest;
+import com.hedera.services.bdd.junit.HapiTestLifecycle;
+import com.hedera.services.bdd.junit.OrderedInIsolation;
+import com.hedera.services.bdd.junit.support.TestLifecycle;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Tag;
+
+@Tag(CRYPTO)
+@OrderedInIsolation
+@HapiTestLifecycle
+@DisplayName("Node Operator Queries")
+/**
+ * A class with Node Operator Queries tests
+ */
+public class AsNodeOperatorQueriesTest extends NodeOperatorQueriesBase {
+
+    @Nested
+    @DisplayName("verify payer balance")
+    /**
+     * A class with Node Operator tests that verify the payer balance
+     */
+    class PerformNodeOperatorQueryAndVerifyPayerBalance {
+
+        @BeforeAll
+        static void beforeAll(@NonNull final TestLifecycle lifecycle) {
+            lifecycle.doAdhoc(createAllAccountsAndTokens());
+        }
+
+        @HapiTest
+        final Stream<DynamicTest> nodeOperatorQueryVerifyPayerBalanceForAccountBalance() {
+            // declare payer account balance variables
+            final AtomicLong currentPayerBalance = new AtomicLong();
+            final AtomicLong payerBalanceAfterFirstQuery = new AtomicLong();
+            final AtomicLong payerBalanceAfterSecondQuery = new AtomicLong();
+            return hapiTest(withOpContext((spec, log) -> {
+                final var initialBalance = getAccountBalance(PAYER).exposingBalanceTo(currentPayerBalance::set);
+                // perform paid query, pay for the query with payer account
+                // the grpc client performs the query to different ports
+                final var firstQuery =
+                        getAccountBalance(NODE_OPERATOR).payingWith(PAYER).signedBy(PAYER);
+                final var waitingForQueryPaymentToReachConsensus = sleepFor(3_000);
+                final var newBalance = getAccountBalance(PAYER).exposingBalanceTo(payerBalanceAfterFirstQuery::set);
+                // perform free query to local port with asNodeOperator() method
+                final var secondQuery = getAccountBalance(NODE_OPERATOR)
+                        .payingWith(PAYER)
+                        .signedBy(PAYER)
```

Comment:
Each `.signedBy(PAYER)` shouldn't be necessary if I remember right. Essentially, I thought the framework could figure out the need to sign with `PAYER` if `.payingWith(PAYER)` is used.

Reply:
Yes, you are absolutely right, I checked that and will remove all .signedBy